### DISCUSSION TITLE
Please put a date on each piece of emulated hardware

### DISCUSSION THREAD

**Discussion by robman501a** [86Box/86Box#3138] [Ideas]
For example, if I want to create a virtual machine from 1992 (like my 486 project), I don't wat to add a part from 1994.
I rather have it this way.
All parts from 1991 or 1992. Nothing greater than '92.

Motherboard
CPU
Maybe add the RAM type to make it more "realistic?"
Video Card
Sound Card
Network Card (N/A. We never owned one)
Hard Drive (How large where hard drives as of that year and what was the bus type?)
CD-ROM (What was the max speed of that year and the bus type?)

Put it this way, if we had the computer in the house in 1992, it wouldn't make any sense to have a motherboard from 1993 or a video card from 1994. ;)

**Comment by Torinde** [comment]
Yes, some extra information would be good to show - in the documentation webpage or in the UI itself (hover tooltips or otherwise) - some of the following?
- year of release
- years of being manufactured
- link to Wikipedia or other description page
- main distinguishing parameters/capabilities
- compatibility notes regarding other hardware and software
- emulation notes
- picture(s)
- etc.

It's nice how 86box by default starts in the first IBM PC configuration. Would be good to be able to select some other pre-configured combinations of emulated hardware as well (XT, AT, "Year 1989 gaming PC", etc.)

Currently I assume it's difficult for a person without knowledge of the PC history to orient himself in all the hardware selections.
